### ISSUE TITLE
package maintained? close old issues?

### ISSUE THREAD

**Issue by protobits** [mank319/elementaryPlus#464]
Hi, I don't really understand if this package is maintained or not. I see issues dating back to 2015 which are still open, yet some associated PRs are merged. Maybe it is a matter of cleanup? Or is the main maintainer away?

**Comment by arshubham** [comment]
@mank319 @quassy @btd1337 What do you think about merging elementaryPlus & https://github.com/btd1337/urutau-icons into one for unified development? I don't design icons but I wan't to help maintain this as much as I can. As in order for eventual release in appcenter having an app window is mandatory I can begin development on that. I have some ideas about that. I like how android icon developer do this for example:

![photo_2018-03-05_18-51-09](https://user-images.githubusercontent.com/14235191/36977185-4fa0bbe4-20a6-11e8-8b01-83636f3011f2.jpg)
![photo_2018-03-05_18-51-12](https://user-images.githubusercontent.com/14235191/36977186-4fe07478-20a6-11e8-8053-88d35d3f58f5.jpg)
![photo_2018-03-05_18-51-14](https://user-images.githubusercontent.com/14235191/36977187-50280e8c-20a6-11e8-9b2f-c9784b0fafb0.jpg)

What do you all think?

**Comment by btd1337** [comment]
@arshubham what can I do?

**Comment by arshubham** [comment]
@btd1337 Can you unify both the themes in a single repo? You can choose any one of them to proceed, but I think this repo will be better suited due to name recognition. Ask @quassy for more info  if you want to maintain this repo. I will work on the front end app this weekend. Also it would be better if you can adapt Papirus install script for the repo https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/blob/master/install.sh  With the install script the development of app can worked upon in a separate repo and can be easily adopted for other icon themes.

I plan to include following features in the application:
1. Download and install latest version of the theme from install script.
2. Apply the theme.
3. Restore the theme.
4. Remove the theme.
5. List and show all icons with names in a searchable format.
6. Icon Request from the app.
7. Show changelog

**Comment by btd1337** [comment]
@arshubham Yes, I can.

But I think that this project (Elementary Plus) not is more mantained. So much that you haven't had any answers yet.

All Elementary Plus icons I have in the Urutau Icons. What do you say we use it there?

Unless the owner of this repository manifests itself, this is not likely.

**Comment by arshubham** [comment]
@btd1337 Fine with me. Can you please look into making the install script. In the meanwhile I am trying to make the application using Papirus theme as a reference and am making it in such a way that it can be ported over to other icon themes with minimal configuration. 
Will continue this conversation on https://github.com/btd1337/urutau-icons from now on.

I will have something ready in a day or two.

**Comment by btd1337** [comment]
Right. Also consider looking at this project: https://github.com/btd1337/urutau-icons-app
